config/routes.rb

```diff
@@ -1,12 +1,18 @@
 Rails.application.routes.draw do
   devise_for :users
-  
-  resources :foods, only: [:index, :new]
-  resources :recipes, only: [:index, :show]
-  resources :public_recipes, only: [:index]
-  resources :inventories, only: [:index, :show, :new, :create, :destroy] do
-    resources :inventory_foods, only: [:create, :destroy]
-  end
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  root 'foods#index'
+  # Defines the root path route ("/")
+  # root "articles#index"
+  root 'food#index'
+  devise_scope :user do
+    get '/user/sign_out' => 'devise/sessions#destroy'
+  end
+  resources :food, only: [:index, :new, :create, :destroy]
```

Comment:
As I mentioned above, for naming conventions, it should `foods` instead of food and `users` instead of `user`.